app/src/main/java/com/into/websoso/ui/main/feed/FeedFragment.kt

```
@@ -145,18 +148,20 @@ class FeedFragment : BaseFragment<FragmentFeedBinding>(fragment_feed) {
+                ?.let { feed ->
+                    EditFeedModel(
+                        feedId = feed.id,
+                        novelId = feed.novel?.id,
+                        novelTitle = feed.novel?.title,
+                        isSpoiler = feed.isSpoiler,
+                        isPublic = feed.isPublic,
+                        feedContent = feed.content,
+                        feedCategory = feed.relevantCategoriesByKr,
+                        imageUrls = feed.imageUrls,
+                    )
+                } ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)